src/main/java/de/nordgedanken/auto_hotkey/ide/highlighter/AhkColors.kt

```diff
@@ -0,0 +1,44 @@
+package de.nordgedanken.auto_hotkey.ide.highlighter
+
+import com.intellij.openapi.editor.DefaultLanguageHighlighterColors
+import com.intellij.openapi.editor.colors.TextAttributesKey
+import com.intellij.openapi.options.colors.AttributesDescriptor
+
+enum class AhkColor(humanName: String, default: TextAttributesKey? = null) {
+    LINE_COMMENT("Comments//Line Comments", DefaultLanguageHighlighterColors.LINE_COMMENT),
+//    FUNCTION("Functions//Function declaration", DefaultLanguageHighlighterColors.FUNCTION_DECLARATION),
```

Comment:
Just remove the commented lines. Thats the beauty of version control. No need to keep them. We can always look at old commits